sketch2Code.sketchplugin

```diff
@@ -79,6 +81,9 @@ function templateBinding(targetView)
     var inits = [NSMutableArray array]
     var addSubviews = [NSMutableArray array]
     var layouts = [NSMutableArray array]
+    layersToSave = [NSMutableArray array]
```

Comment:
@csacsi I think this should be at line 39, otherwise layersToSave won't be initialised if `templateBinding` is not called